### PR TITLE
962 events

### PR DESCRIPTION
**Pull request by helenb**
Events styling, and I have also moved the js from the template into separate js files. Note that I have recreated styles for now even where they are similar - we may be able to refactor if time later.

### CODE REVIEW

File: django-verdant/rca/static/rca/css/streamfield-blocks/events.less [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    }
+
+    &__dates {
+        position: absolute;
+        top: 0;
+        left: 0;
+        background-color: #A4429C; // design doesn't match existing colour palette
+        .opacity(90);
+        color: @color-white;
+        padding: @spacing-half @spacing-one @spacing-half @spacing-half;
```
Comment (hippogriffic) [inline django-verdant/rca/static/rca/css/streamfield-blocks/events.less]: I feel like
```
padding: @spacing-half;
padding-right: @spacing-one;
```
would be more readable

File: django-verdant/rca/static/rca/css/streamfield-blocks/events.less [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+        position: relative;
+        margin-bottom: @spacing-one;
+    }
+
+    &__dates {
+        position: absolute;
+        top: 0;
+        left: 0;
+        background-color: #A4429C; // design doesn't match existing colour palette
+        .opacity(90);
```
Comment (hippogriffic) [inline django-verdant/rca/static/rca/css/streamfield-blocks/events.less]: I noticed you were using this before, is there a reason to not just use `opacity: 0.9;`?

File: django-verdant/rca/static/rca/js/showcase-block.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+$(function(){
+    // Showcase carousel
+    var $showcaseSlider = $('.js-showcase');
+    var prevButton = '<button aria-label="Previous" class="showcase__prev showcase__button" type="button"><svg class="showcase__arrow"><title>Previous</title><use xlink:href="#chevron"></use></svg></button>';
+    var nextButton = '<button aria-label="Next" class="showcase__next showcase__button" type="button"><svg class="showcase__arrow"><title>Previous</title><use xlink:href="#chevron"></use></svg></button>';
```
Comment (hippogriffic) [inline django-verdant/rca/static/rca/js/showcase-block.js]: in my branch I made some adjustments to this markup to make them more generic so I could reuse the buttons and styles on the testimonial block. We should probably discuss the best approach with these. I'll split the js in my branch out into separate files to match what you've done here

File: django-verdant/rca/static/rca/css/streamfield-blocks/events.less (hunk 2 above)
Comment (helenb) [reply django-verdant/rca/static/rca/css/streamfield-blocks/events.less]: The mixin adds the filter fallback, as when the site was first built we were including support for older browsers. I don't think there's any harm in continuing to use it for consistency.

File: django-verdant/rca/static/rca/js/showcase-block.js (hunk 3 above)
Comment (helenb) [reply django-verdant/rca/static/rca/js/showcase-block.js]: thanks and yes let's have a chat - will check your testimonial branch next.

File: django-verdant/rca/static/rca/css/streamfield-blocks/events.less (hunk 1 above)
Comment (helenb) [reply django-verdant/rca/static/rca/css/streamfield-blocks/events.less]: Updated.

**Comment by helenb** [comment]
Are you happy for me to merge this now Beth?

**Pull request merged**